### DISCUSSION TITLE
Dates don't render correctly

### DISCUSSION THREAD

**Discussion by juandspy** [nunocoracao/blowfish#1034] [Q&A] [answered]
I've just created a [site](https://juandspy.github.io/posts/) and the dates are not rendered correctly. See:

<img width="699" alt="image" src="https://github.com/nunocoracao/blowfish/assets/42124482/a27eadbc-79c0-46b8-92f0-e33392f49930">

When running `hugo serve` locally it renders correctly, so it has to be something wrong with the GH page settings (?)

**Comment by AustinCasteel** [comment]
I am having the same issue, where all post meta data is displaying the html as plain text instead of actually rendering the html across the whole site.

**Comment by myseq** [comment, marked as answer]
The latest Hugo version, 0.120.x onward, is not compatible with Blowfish. Need to downgrade back to 0.119 (rev 17367); or refresh the Blowfish version with "$ git submodule update --remote --merge".

**Reply by juandspy** [reply]
Thanks a lot! @AustinCasteel , take a look at this fix :)

**Reply by AustinCasteel** [reply]
I was starting to think that was the issue as I continued on and I kept running into other issues. Updating blowfish did resolve the meta data issue, though the other issues I have run into still persist. Will make another post about them.

**Comment by nunocoracao** [comment]
@juandspy latest version should support that breaking change in Hugo
